MEME/MEME/Model/ModelHomeView/View/ModelHomeViewController.swift

```diff
@@ -0,0 +1,403 @@
+//
+//  ViewController.swift
+//  MEME
+//
+//  Created by 이동현 on 1/9/24.
+//
+
+import UIKit
+import SnapKit
+import RxSwift
+
+final class ModelHomeViewController: UIViewController {
+    // MARK: - UIProperty
+    private let scrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        scrollView.showsHorizontalScrollIndicator = false
+        scrollView.showsVerticalScrollIndicator = false
+    
+        return scrollView
+    }()
+    private let contentsView = UIView()
+    private var memeLogoImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "logo")
```

Comment:
```suggestion
        imageView.image = .logo
```
와 같이 사용하면 어떨까요? 이미지의 이름이 잘못되었거나 해당 이미지가 없으면 옵셔널 값을 가져오게 되는데, 이런 문제를 예방할 수 있을 것 같습니다.